secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -104,6 +109,27 @@ def secret_id(
         # Secret was already deleted, probably in the test
         print(f"Secret {secret_id} was not found.")
 
+@pytest.fixture()
+def regional_secret_version(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    regional_secret: Tuple[str, str, str, str],
+) -> Iterator[Tuple[str, str, str, str, str]]:
```

Comment:
Use a dataclass to describe what the tuple of 4/5 items are.

Reply:
Fixed this in : https://github.com/GoogleCloudPlatform/python-docs-samples/pull/12480 ( child branch) for all the samples to avoid merge conflicts.